contracts/network/BancorNetwork.sol

```diff
@@ -523,12 +533,13 @@ contract BancorNetwork is IBancorNetwork, Upgradeable, ReentrancyGuardUpgradeabl
 
             // request the pool migrator to migrate the pool and get the new pool collection it exists in
             IPoolCollection newPoolCollection = _poolMigrator.migratePool(pool);
-            if (newPoolCollection == IPoolCollection(address(0))) {
-                continue;
-            }
+            IPoolCollection prevPoolCollection = _collectionByPool[pool];
 
             // update the mapping between pools and their respective pool collections
             _collectionByPool[pool] = newPoolCollection;
+
+            emit PoolAdded(pool, newPoolCollection);
+            emit PoolRemoved(pool, prevPoolCollection);
```

Comment:
Removed should be triggered before Added

Reply:
👍

Reply:
👍